snow/consensus/snowball/tree.go

```diff
@@ -17,33 +17,34 @@ var (
 	_ node      = (*binaryNode)(nil)
 )
 
-func NewTree(params Parameters, choice ids.ID) Consensus {
-	snowball := newUnarySnowball(params.BetaVirtuous)
+func NewTree(cf ConsensusFactory, params Parameters, choice ids.ID) Consensus {
```

Comment:
Can we just spell this out as `consensusFactory` instead of `cf`?

Reply:
Yup